src/libraries/Microsoft.PowerFx.Core/Localization/StringResources.cs

```diff
@@ -108,9 +109,11 @@ internal class PowerFxStringResources : IExternalStringResources
 
         internal string ResourceLocation => _resourceManager.ResourceLocation;
 
-        [ThreadSafeProtectedByLockAttribute("_errorResources")]
+        [ThreadSafeProtectedByLock("_lock")]
         private static readonly Dictionary<string, Dictionary<string, ErrorResource>> _errorResources = new (StringComparer.OrdinalIgnoreCase);
```

Comment:
Also concerned that this field is static.  But everybody holding onto the containing class here (PowerFxStringSources)  is static, so making it instance doesn't really buy us much. We should followup on this ...